dockertest/images_unittests.py

```diff
@@ -341,6 +341,15 @@ def test_addr_no_user(self):
         self.assertEqual(user, None)
         self.assertEqual(addr, "address:1234")
 
+    def test_empty_tag(self):
+        test = "myname:"   # note the trailing colon with no actual tag name
```

Comment:
The context here is support for docker-1.9 events output. In particular, the 1.9 output format is:

    <timestamp> <identifier>: <action>

...where <identifier> can be a SHA or a symbolic name. The symbolic name is usually of the form foo/bar:tag or even bar:tag but in some cases tag is null:

    2016-04-11T12:17:25.671704251-04:00 change_repository_bqtg:: tag
    2016-04-11T12:17:33.963279935-04:00 double_tag_ndal:: tag

(note the double colons: one of those is part of the docker-events output format, the other one is an empty tag). And, FWIW, docker-1.9 allows specifying an image name with a trailing colon (docker-1.10 does not).

Should I add a FIXME comment to the + -> * change, indicating that it should be reverted back to + once docker-1.10 is the standard?